packages/ddp-client/livedata_connection_tests.js

```diff
@@ -95,6 +95,92 @@ Tinytest.add("livedata stub - receive data", function (test) {
   test.isUndefined(conn._updatesForUnknownStores[coll_name]);
 });
 
+Tinytest.add("livedata stub - buffering data", function (test) {
+  var lolex = Npm.require('lolex');
```

Comment:
@tmeasday Ok, I just didn't see *any* (???_ other internal packages doing anything besides `Npm.depends` and `Npm.require` for actual npm packages so wasn't sure if it was supported or desired.  Do I need to create `package.json` files for `ddp-client` if I go that route?